content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: blog
+title: "Gateway API v1.1: Service mesh, GRPCRoute, and a whole lot more"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+author: >
+  [Richard Belleville](https://github.com/gnossen) (Google),
+  [Frank Budinsky](https://github.com/frankbu) (IBM),
+  [Arko Dasgupta](https://github.com/arkodg) (Tetrate),
+  [Flynn](https://github.com/kflynn) (Buoyant),
+  [Candace Holman](https://github.com/candita) (Red Hat),
+  [John Howard](https://github.com/howardjohn) (Solo.io),
+  [Christine Kim](https://github.com/xtineskim) (Isovalent),
+  [Mattia Lavacca](https://github.com/mlavacca) (Kong),
+  [Keith Mattix](https://github.com/keithmattix) (Microsoft),
+  [Mike Morris](https://github.com/mikemorris) (Microsoft),
+  [Rob Scott](https://github.com/robscott) (Google),
+  [Grant Spence](https://github.com/gcs278) (Red Hat),
+  [Shane Utt](https://github.com/shaneutt) (Kong),
+  [Gina Yeh](https://github.com/ginayeh) (Google),
+  and other review and release note contributors.
```

Comment:
```suggestion
  and other review and release note contributors
```

If we can make this smaller, it'll help I think. Some people will stop reading before they reach the end of this list.

Reply:
I'd like to keep it as is for now.  Next time, we'll talk about this in our community gathering prior to creating the PR.